source/features/cross-deleted-pr-branches.tsx

```diff
@@ -0,0 +1,62 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+
+function inPR(): void {
+	let deletedBranch: string | undefined;
+	const lastBranchAction = select.all(`
+		.discussion-item-head_ref_deleted .commit-ref,
+		.discussion-item-head_ref_restored .commit-ref
+    `).pop();
+
+	if (lastBranchAction && lastBranchAction.closest('.discussion-item-head_ref_deleted')) {
+		deletedBranch = lastBranchAction.textContent!.trim();
+	}
+
+	// Making deleted branch link to the default fork
+	for (const el of select.all('.commit-ref[title].head-ref[title]')) {
+		const [repo] = el.title.split(':', 1);
+		const branchName = el.textContent!.trim();
+		if (branchName !== 'unknown repository') {
+			if (branchName === deletedBranch) {
+				el.children[0].setAttribute('href', `${branchName}/${repo}`);
```

Comment:
Use `.firstChild.href = `

Also I don't think it's `{branchName}/${repo}`

Reply:
Doing something like `el.children[0].firstChild.href` doesn't works for me. Isn't pointing just to the repo will load the default branch?

Reply:
https://github.com/Microsoft/TypeScript/issues/16920 This helped and I made it work, somehow.